angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -64,12 +64,18 @@ import {
           </sprk-link>
         </div>
 
-        <ul class="sprk-c-Dropdown__links">
+        <ul
+          class="sprk-c-Dropdown__links"
+          role="listbox"
+          [attr.aria-label]="title ? title : screenReaderText"
```

Comment:
same here, if title or screenReaderText aren't there, some kind of default should be used.